README.rst

```diff
@@ -119,7 +112,7 @@ portable version you should observe the following install steps.*
 
        > easy_install openslides-x.x.tar.gz
```

Comment:
We should recommend `pip` instead of `easy_install`. I am not sure but I think first we have to install pip:
`easy_install pip`. And then: `pip install openslides-x.x.tar.gz`